curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646baf7968b91006bc1856c9.md

```diff
@@ -0,0 +1,145 @@
+---
+id: 646baf7968b91006bc1856c9
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+In the same class selector named `.select-icon` in the CSS file, add the following properties:
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed